src/rpc/handlers/NFTHistory.cpp

```diff
@@ -47,7 +47,7 @@ NFTHistoryHandler::process(NFTHistoryHandler::Input input, Context const& ctx) c
     }
 
     if (minIndex > maxIndex)
-        return Error{Status{RippledError::rpcINVALID_PARAMS, "invalidIndex"}};
+        return Error{Status{RippledError::rpcLGR_IDXS_INVALID, "invalidIndex"}};
```

Comment:
Let us remove "invalidIndex" error message. It will use the ripped specified error message: "Ledger indexes invalid."